test/lit/passes/coalesce-locals-eh-legacy.wast

```diff
@@ -13,10 +15,6 @@
     (i32.const 1984)
   )
 
- (tag $e)
-
- (tag $any (param (ref any)))
-
```

Comment:
I wonder what caused this change?

Reply:
Yeah, thanks, that's a good change I think.